src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/InntektsmeldingRouting.kt

```diff
@@ -44,7 +49,9 @@ private fun Route.innsending(services: Services) {
             val request = call.receive<InntektsmeldingRequest>()
             val sluttbrukerOrgnr = tokenValidationContext().getSystembrukerOrgnr()
             val lpsOrgnr = tokenValidationContext().getConsumerOrgnr()
-
+            if (!tokenValidationContext().harTilgangTilRessurs(IM_RESSURS)) {
```

Comment:
Et annet alternativ er å throwe error og catche den for alle routes slik vi gjør i simba feks? 
Men det blir jo egentlig det samme som her så tenker dette er bra 👍 .

Reply:
Ja, det er absolutt en mulighet å heller gjøre det - vi kan godt teste å skrive om senere, for å se om det blir enklere / mindre kode